aks-kubesphere-linux/parameters.md

```diff
@@ -0,0 +1,33 @@
+parameters
```

Comment:
update this file after removing the defaultValues for usernames/passwords

Reply:
yeah, I will change this.